content/fast/use-imagemin-to-compress-images/codelab-imagemin-grunt.md

```diff
@@ -10,24 +10,15 @@ web_published_on: 2018-11-05
 glitch: imagemin-grunt
 ---
 
-## Install the Imagemin Grunt plugin
+## Setup the Imagemin Grunt plugin
 
-This Glitch already contains `grunt` and `grunt-cli` for you, but you'll need to
-install `grunt-contrib-imagemin`.
+This Glitch already contains `grunt`, `grunt-cli`, and the `grunt-contrib-imagemin` 
+plugin. To add the configuration for Imagemin, you'll need to edit your 
+`gruntfile.js` file.
 
 - Click the **Remix to Edit** button to make the project editable.
```

Comment:
@kjue25 What do you think about switching to numbered steps anytime there is more than 1 step?

Reply:
Yeah I like that - especially since some of the bullet points wouldn't count as steps in the setup process.  I can update those.